oma-console/src/pager.rs

```diff
@@ -107,6 +108,28 @@ enum PagerInner {
     Finished(Vec<String>),
 }
 
+///
+/// `OmaPager` is a structure that represents a pager displaying content in a terminal UI.
+///
+/// # Fields
+///
+/// * `inner` - Represents the internal state of the pager, which can be either `Working` or `Finished`.
```

Comment:
Write the comment on top of the field, for example:

```rust
pub struct OmaPager<'a> {
    /// Represents the internal state of the pager, which can be either `Working` or `Finished`.
    inner: PagerInner,
    /// The state of the vertical scrollbar.
    vertical_scroll_state: ScrollbarState,
    /// The state of the horizontal scrollbar.
    horizontal_scroll_state: ScrollbarState,
    ...
```

Only helps the IDE read field comments.

Reply:
Fixed in https://github.com/AOSC-Dev/oma/pull/228/commits/b5856e7e5ff0dc37abdf09b60dae02a101049901